### PR TITLE
Support for Boolean Properties

### PR DESCRIPTION
**Pull request by kristoferbaxter**
Support Transferring Boolean Properties.

There are no tests for this part of the codebase, so need to add a lot of them before merging.

### CODE REVIEW

File: src/main-thread/worker.ts [hunk 1]
```diff
@@ -72,7 +73,7 @@ export class WorkerContext {
         ${authorScript}
       }).call(WorkerThread.workerDOM);
   //# sourceURL=${encodeURI(config.authorURL)}`;
-    this.worker = new Worker(URL.createObjectURL(new Blob([code])));
+    this[TransferrableKeys.worker] = new Worker(URL.createObjectURL(new Blob([code])));
```
Comment (dvoytenko) [inline src/main-thread/worker.ts]: A bit confused. Why this?

File: src/worker-thread/dom/Element.ts [hunk 2]
```diff
@@ -498,30 +498,32 @@ export class Element extends ParentNode {
     };
 
     return new Promise(resolve => {
+      const messageHandler = ({ data }: { data: MessageToWorker }) => {
+        if (
+          data[TransferrableKeys.type] === MessageType.GET_BOUNDING_CLIENT_RECT &&
+          (data as BoundingClientRectToWorker)[TransferrableKeys.target][0] === this[TransferrableKeys.index]
+        ) {
+          removeEventListener('message', messageHandler);
```
Comment (dvoytenko) [inline src/worker-thread/dom/Element.ts]: Bad merge? I thought I already reviewed this in the transferrables PR...

File: tools/linter/.babelrc [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (dvoytenko) [inline tools/linter/.babelrc]: Bad merge as well?

**Comment by kristoferbaxter** [comment]
This PR appears to be incorrectly marked as merging to something besides master. Sorry, one minute.

**Pull request closed without merging**